ehrapy/preprocessing/_bias.py

```diff
@@ -0,0 +1,119 @@
+from collections.abc import Iterable
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+
+from ehrapy import logging as logg
+from ehrapy.anndata import anndata_to_df
+
+
+def bias_detection(
+    adata: AnnData,
+    sensitive_features: Iterable[str] | Literal["all"],
```

Comment:
Consider adding a `*` after `sensitive_features` to force the rest to be keyword arguments